x-pack/plugins/threat_intelligence/public/containers/integrations_guard/integrations_guard.tsx

```diff
@@ -7,26 +7,53 @@
 
 import { EuiLoadingLogo } from '@elastic/eui';
 import React, { FC } from 'react';
+import { useRequest, UseRequestConfig } from '@kbn/es-ui-shared-plugin/public';
+import { HttpSetup } from '@kbn/core/public';
 import { EmptyPage } from '../../modules/empty_page';
-import { useIndicatorsTotalCount } from '../../modules/indicators';
 import { SecuritySolutionPluginTemplateWrapper } from '../security_solution_plugin_template_wrapper';
 
+let httpClient: HttpSetup;
+
+const PACKAGES_CONFIG: UseRequestConfig = {
+  path: '/api/fleet/epm/packages',
+  method: 'get',
+};
+
+const THREAT_INTEL_CATEGORY = 'threat_intel';
+
+const installationStatuses = {
```

Comment:
I stole this object from the `fleet` plugin, again to get things working. I will dig more to see if we can retrieve it from somewhere instead of hardcoding it here